spec/integrations/posts_integrations_spec.rb

```diff
@@ -0,0 +1,128 @@
+    end
+
+    it 'displays the posts title' do
+      expect(page).to have_content('Post 1')
+    end
+
+    it 'displays some of the posts body' do
+      expect(page).to have_content('Line one. Line two.')
+      expect(page).not_to have_content('Line three, Line four. Line five')
+    end
+
+    it 'displays the recent 5 comments of each post' do
+      expect(page).to have_content('Comment 1')
+      expect(page).to have_content('Comment 2')
+      expect(page).to have_content('Comment 3')
+      expect(page).to have_content('Comment 4')
+      expect(page).to have_content('Comment 5')
+      expect(page).not_to have_content('Comment 6')
+    end
+
+    it 'displays the number of comments each post has' do
+      expect(page).to have_content('View 6 previous comments')
+    end
+
+    it 'displays the number of likes each post has' do
+      expect(page).to have_content('2')
+    end
+
+    it 'has pagination if there are more posts than fit the view' do
+      click_link('Next')
+      expect(page).to have_content('Post 3')
+      expect(page).to have_content('Post 4')
+
+      expect(page).not_to have_content('Post 1')
+      expect(page).not_to have_content('Post 2')
+    end
+  end
+
+  context '#posts index click' do
+    before(:example) do
+      @user1 = User.create(id: 1, name: 'Brad', photo: 'https://randomuser.me/api/portraits/men/65.jpg',
+                           bio: 'I am user 1')
+      @post1 = Post.create(id: 1, author: @user1, title: 'Post 1', text: 'Text 1', created_at: Time.now - 1.day)
+      visit user_path(1)
+    end
+
+    it 'redirects to a posts show page when clicked' do
+      click_link('Continue reading')
+      expect(page).to have_current_path(user_post_path(1, 1))
+    end
+  end
+
+  describe '#posts show' do
+    before(:example) do
+      @user1 = User.create(id: 1, name: 'Brad', photo: 'https://randomuser.me/api/portraits/men/65.jpg',
+                           bio: 'I am user 1')
+      @post1 = Post.create(id: 1, author: @user1, title: 'Post 1', text: 'Hello world')
+      6.times do |i|
+        Comment.create(id: i + 1, author: @user1, post: @post1, text: 'Comment 1')
+      end
+
+      2.times do |i|
+        Like.create(id: i + 1, author: @user1, post: @post1)
+      end
+
+      visit user_post_path(1, 1)
+    end
+
+    it 'displays the posts title' do
+      expect(page).to have_content('Post 1')
+    end
+
+    it 'displays who wrote the post' do
+      expect(page).to have_content('Post 1 #1, by Brad')
+    end
+
+    it 'displays the number of likes the post has' do
+      expect(page).to have_content('Likes: 2')
+    end
+
+    it 'displays the posts body' do
+      expect(page).to have_content('Hello world')
+    end
+
+    it 'displays the username of the author of each comment' do
+      expect(page).to have_content('Brad: Comment 1')
+    end
+
+    it 'displays each comment made' do
+      expect(page).to have_content('Comment 1')
+    end
+  end
+end
```

Comment:
- [ ] Hi @XsidX, all your tests are failing. Please kindly fix the issue. The screenshot below :point_down: 


![Screenshot from 2022-11-01 16-33-53](https://user-images.githubusercontent.com/63087491/199273965-b23034aa-6415-4193-b7e7-0754d8b29e94.png)

Reply:
Hello, @FrederickMih 
- When working with tailwindcss as I am in this project, you need to build the tailwindcss assets atleast once after cloning the project.
 - The assets are compiled automatically when the rails server is started but for the tests, they have to be build manually with this command `rails assets:precompile`, atleast once after cloning.

Reply:
Hi @XsidX, I don't think the issue is at the level of tailwindcess. I already ran the command rails assets:precompile

Reply:
> Hi @XsidX, I don't think the issue is at the level of tailwindcess. I already ran the command rails assets:precompile

Hello @FrederickMih, I would be happy to hope on a zoom call to explain this.